AllJoyn/Platform/Bridge/BridgeCore/AllJoynHelper.cpp

```diff
@@ -1242,6 +1359,10 @@ QStatus AllJoynHelper::GetSignature(_In_ PropertyType propertyType, _Out_ std::s
         signature = "as";
         break;
 
+    case PropertyType::StringDictionary:
+        signature = "a{ss}";
```

Comment:
"a{ss}" is actually correct. AllJoyn (DBus) represents dictionaries as an array of key-value pairs.

---
In reply to: [67949524](https://github.com/tjaffri/msiot-samples/pull/7#discussion_r67949524) [](ancestors = 67949524,67949261)